src/ILOPoolSale.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
let's make it MIT

Reply:
I think all of contracts are under BUSL-1.1 license?